src/org/impact2585/frc2016/systems/WheelSystem.java

```diff
@@ -181,86 +145,51 @@ public void initializeForwardPID(double setpoint) {
 	 * @param rotate how much the robot will turn
 	 */
 	public void drive(double driveForward, double rotate) {
-		drivetrain.arcadeDrive(driveForward, -rotate);
+		drivebase.arcadeDrive(driveForward, -rotate);
```

Comment:
If you passed true into the constructor of `Drivetrain`, would this mean that you're inverting rotation twice?